pkg/util/remotecommand/constants.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
perhaps pkg/util/remotecommand should be in apimachinery, i see similar comment for port forwarding from @deads2k